assets/keymaps/default-macos.json

```diff
@@ -432,6 +432,7 @@
     "context": "Editor",
     "bindings": {
       "ctrl-j": "editor::JoinLines",
+      "cmd-shift-j": "editor::JoinLines",
```

Comment:
I'm not a Mac user so these might be the norm, but they do stand out as dupes.

Reply:
I don't have a strong feeling here.  There was enough ambiguity that I figured it might be worth duplicating to have Zed consistent across platforms. I can be convinced otherwise.

Sublime currently uses `cmd+shift+j` by default now (2024).

```
// Sublime Linux
{ "keys": ["ctrl+shift+j"], "command": "join_lines" },
// Sublime Mac
{ "keys": ["super+shift+j"], "command": "join_lines" },
```
See: https://gist.github.com/notpeter/0c263c8fd6e50a67aa2e4f36e4743d63

Atom MacOS [source](https://github.com/atom/atom/blob/master/keymaps/darwin.cson#L181)
```
'cmd-j': 'editor:join-lines'
```

Atom Linux [source](https://github.com/atom/atom/blob/master/keymaps/linux.cson#L129)
```
'ctrl-j': 'editor:join-lines'
```

When Atom added it 10+ years ago they claimed to be matching `cmd+j` from Sublime...totally possible SublimeText switched from `cmd+j` to `cmd+shift+j` at some point or it was originally transposed in error. 

JetBrain MacOS
<img width="361" alt="image" src="https://github.com/zed-industries/zed/assets/145113/5a0295c2-9a32-4c26-a966-7daf3a5bef02">

JetBrains Windows/Linux
<img width="436" alt="Screenshot 2024-07-11 at 0 12 37" src="https://github.com/zed-industries/zed/assets/145113/0163ae0a-1794-4cb4-aa37-ba7e6497edd6">

Earlier atom used `meta-j` (`alt-j`) and emacs uses `M-^` for `join-line`.